Form/Extension/DisableCsrfExtension.php

```diff
@@ -95,6 +96,6 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getExtendedType()
     {
-        return 'form';
+        return FormType::class;
```

Comment:
It doesn't work with PHP >= 5.5 and we still support PHP 5.3. Can you use the class name as a string instead please?